documents/motions/10.2024.0 - Committees and Teams.md

```diff
@@ -1,51 +1,55 @@
 # SUBMISSION OF PROPOSED MOTION
 
-**Motion number:** 10.2022.0  
+**Motion number:** 10.2024.0  
 **Subject:** WCA Committees and Teams  
 **Intent:** Definition of WCA Committees and Teams  
 **Submitted by:** WCA Board  
-**Date:** May 1, 2022
+**Date:** August 1, 2024
 
 # Motion
 
 The WCA will have Committees and/or Teams that have the role of Advisory Committee as described in the Bylaws.
 
 1. The Committees and Teams are responsible for operational activities of the WCA. Any reference to a Team should be understood to also be a reference to a Committee and vice versa.
 2. All WCA Committees and Teams shall be formed for an indefinite period, unless otherwise decided by the WCA Board.
+   1. If a WCA Committee has fewer than 2 members it will not be considered constituted and its rights and duties will be undertaken by the WCA Board until 2 or more members are appointed. 
 3. Each WCA Committee and Team will have a direct relationship with at least one Director for management, communication, and escalation.
-4. Each WCA Committee and Team shall have one Leader.
+4. Each Committee or Team may split into one or more sub-teams specialized in handling one or more of the tasks of the Committee or Team.
```

Comment:
It makes sense to be able to do this of course, but the motions did not previously prohibit it. To me it doesn't seem necessary to include unless the subteams were official designations

Reply:
I agree on it's own it would not be relevant to define sub-teams however this definition was added so that we can make the clarifying statement in line 54 of this document regarding Senior Members. This was added to clarify some questions and concerns WCT had with the WCT-China team